File: MinorityReport/MinorityReport/ViewController.m
//
//  ViewController.m
//  MinorityReport
//
//  Created by Peter Hitchcock on 10/2/14.
//  Copyright (c) 2014 Peter Hitchcock. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@property (weak, nonatomic) IBOutlet UILabel *theFuture;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


- (IBAction)onDrag:(UIPanGestureRecognizer *)panGesture {
    CGPoint point = [panGesture translationInView:self.view];
    self.theFuture.center =
    NSLog(@"what");
}




@end

File: Vehicle/Vehicle/ViewController.swift
//
//  ViewController.swift
//  Vehicle
//
//  Created by Peter Hitchcock on 10/3/14.
//  Copyright (c) 2014 Peter Hitchcock. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        var car = WheeledVehicle()
        car.stop()
        car.numberOfWheels(3)
        car.currentSpeed



        println("\(car.currentSpeed)")

        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

